src/sentry/sentry_metrics/querying/data_v2/execution.py

```diff
@@ -99,160 +96,223 @@ def _build_aligned_seq(
     return aligned_seq
 
 
-@dataclass(frozen=True)
-class ExecutableQuery:
-    with_series: bool
-    with_totals: bool
+def _push_down_group_filters(
```

Comment:
Moved the function here, since it's a generic function that doesn't need to be on the instance.